apis/Google.Cloud.Firestore/Google.Cloud.Firestore.IntegrationTests/MultipleFirestoreDbFixture.cs

```diff
@@ -0,0 +1,63 @@
+// Copyright 2022, Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Apis.Auth.OAuth2;
+using Google.Apis.Http;
+using Google.Cloud.ClientTesting;
+using Newtonsoft.Json;
+using System;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Google.Cloud.Firestore.IntegrationTests;
+
+[CollectionDefinition(nameof(MultipleFirestoreDbFixture))]
+public class MultipleFirestoreDbFixture : CloudProjectFixtureBase, ICollectionFixture<MultipleFirestoreDbFixture>
+{
+    private const string RestApiUrlBase = "https://firestore.googleapis.com/v1";
+    private const string ProjectEnvironmentVariable = "FIRESTORE_TEST_PROJECT";
+    private static readonly HttpClient s_httpClient;
```

Comment:
Given that we only ever need to use this from an instance context, I wonder whether it would make sense to have this as an instance field, and move all the initialization into the constructor. That's *usually* nicer than having a static constructor. It doesn't matter too much though.

Reply:
These are made static to basically share the HTTP client object between multiple instances. Although its a fixture that will be shared by multiple tests. I think it make sense to convert them to instance variables. 
Thanks.